src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -98,10 +98,14 @@ public static DynamoPreferencesData Default()
     /// </summary>
     public struct HostAnalyticsInfo
     {
-        // Dynamo variation identified by host.
+        // Dynamo variation identified by host, e.g. Dynamo Revit
         public string HostName;
         // Dynamo variation version specific to host
         public Version HostVersion;
+        // Dynamo host application name, e.g. Revit
+        public string HostAppName;
+        // Dynamo host application version, e.g. 2025
+        public string HostAppVersion;
```

Comment:
is there a problem with using the `Version` type?

Reply:
I guess it was more like a year name when I started the work, e.g. 2025 but the more I think about it, I think Version works and provides more flexibility

Reply:
How about `HostProductName`  and `HostProductVersion`?